From 68d2d2dfc532c50b4a358baeca08b409a5bb5395 Mon Sep 17 00:00:00 2001
From: Don Zickus <dzickus@redhat.com>
Date: Tue, 16 Sep 2014 17:03:21 -0400
Subject: [usb] wusbcore: fix up line break coding style issues in wa-hc.h

Message-id: <1410887463-20674-130-git-send-email-dzickus@redhat.com>
Patchwork-id: 95496
O-Subject: [RHEL7 PATCH 129/591] USB: wusbcore: fix up line break coding style issues in wa-hc.h
Bugzilla: 1110939
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

https://bugzilla.redhat.com/show_bug.cgi?id=1110939

commit 62316ff4bc619d5c0ac9d08c95e67282162dc93b
Author: Rahul Bedarkar <rahulbedarkar89@gmail.com>
Date:   Sat Jan 4 14:10:24 2014 +0530

    USB: wusbcore: fix up line break coding style issues in wa-hc.h

    Signed-off-by: Rahul Bedarkar <rahulbedarkar89@gmail.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/usb/wusbcore/wa-hc.h b/drivers/usb/wusbcore/wa-hc.h
index 529893f..a2ef84b 100644
--- a/drivers/usb/wusbcore/wa-hc.h
+++ b/drivers/usb/wusbcore/wa-hc.h
@@ -197,10 +197,10 @@ struct wahc {
  struct mutex rpipe_mutex; /* assigning resources to endpoints */
 
  /*
-  * dti_state is used to track the state of the dti_urb.  When dti_state
+  * dti_state is used to track the state of the dti_urb. When dti_state
   * is WA_DTI_ISOC_PACKET_STATUS_PENDING, dti_isoc_xfer_in_progress and
-  * dti_isoc_xfer_seg identify which xfer the incoming isoc packet status
-  * refers to.
+  * dti_isoc_xfer_seg identify which xfer the incoming isoc packet
+  * status refers to.
   */
  enum wa_dti_state dti_state;
  u32 dti_isoc_xfer_in_progress;
@@ -211,7 +211,7 @@ struct wahc {
  void *dti_buf;
  size_t dti_buf_size;
 
- unsigned long dto_in_use; /* protect dto endoint serialization. */
+ unsigned long dto_in_use; /* protect dto endoint serialization */
 
  s32 status;   /* For reading status */
 
-- 
1.7.1